packages/itwinui-react/tsconfig.test.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.json",
+  // TODO: Test all the files, and not just the Table files or just the build files.
```

Comment:
nit: maybe move this todo to issues to not forget about this.

Reply:
Sounds good, created https://github.com/iTwin/iTwinUI/issues/1561